tools/coq_makefile.ml

```diff
@@ -124,6 +124,7 @@ let quote s = if String.contains s ' ' || CString.is_empty s then "'" ^ s ^ "'"
 
 let generate_makefile oc conf_file local_file args project =
   let makefile_template =
+    Format.eprintf "%s@\n%!" Sys.(getcwd ());
```

Comment:
Is this a leftover?

Reply:
Yes sorry, I was trying out to figure what was going on with path handling.

Reply:
Indeed I am very close to factorize path handling a bit across all Coq, however I still need to figure out how to act on installed / built targets.